cli/progress-bar/src/terminal.rs

```diff
@@ -43,7 +43,7 @@ impl TerminalHandle {
 
     pub fn new_enabled() -> Self {
         if superconsole::SuperConsole::compatible() {
-            Self(TerminalHandleInner::new().into())
```

Comment:
This code is already correct https://doc.rust-lang.org/std/option/enum.Option.html#impl-From%3CT%3E-for-Option%3CT%3E